libflux/src/libstd/lib.rs

```diff
@@ -0,0 +1,14 @@
+use flux::semantic::env::Environment;
+use flux::semantic::flatbuffers::semantic_generated::fbsemantic as fb;
+
+use flatbuffers;
+
+pub fn prelude() -> Option<Environment> {
+    let buf = include_bytes!(concat!(env!("OUT_DIR"), "/prelude.data"));
```

Comment:
So `include_bytes!` will put the contents of `${OUT_DIR}/prelude.data` directly into this rust source file?

Reply:
Yes correct.